tool/tsh/tsh.go

```diff
@@ -754,12 +759,31 @@ func executeAccessRequest(cf *CLIConf) {
 	onStatus(cf)
 }
 
-func showNodes(nodes []services.Server, verbose bool) {
+func showNodes(nodes []services.Server, verbose bool, format ...string) {
+	if len(format) != 0 {
+		switch strings.ToLower(format[0]) {
+		case teleport.JSON:
+			out, err := json.MarshalIndent(nodes, "", "  ")
+			if err != nil {
+				utils.FatalError(err)
+			}
+			fmt.Println(string(out))
+			return
+		case teleport.Text:
+			// print out node host names as plain text
+			for _, n := range nodes {
+				fmt.Println(n.GetHostname())
+			}
+			return
+		}
+	}
```

Comment:
Let's have 3 format options:
- (default) `text` or `table` - the asciitable below
- `plain` or `names` or `text` - one node hostname per line
- `json`

That way `format` is always set to something.
Also, any other `format` value should be an error to avoid empty output

Reply:
Should I add 'table' and 'names' as a constant somewhere, like in the teleport package?

Reply:
Yep, right next to `teleport.Text` and `teleport.JSON`

Reply:
The `text` format already means "print a table" everywhere else in our codebase.  We should keep it consistent and use `text` to represent the table option here as well.